dams/src/config/client.rs

```diff
@@ -1,25 +1,11 @@
 use serde::{Deserialize, Serialize};
-use std::{
-    path::{Path, PathBuf},
-    time::Duration,
-};
-
-use crate::defaults::client as defaults;
+use std::path::{Path, PathBuf};
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
 #[non_exhaustive]
 pub struct Config {
-    #[serde(with = "humantime_serde", default = "defaults::connection_timeout")]
-    pub connection_timeout: Option<Duration>,
-    #[serde(default = "defaults::max_pending_connection_retries")]
-    pub max_pending_connection_retries: usize,
-    #[serde(with = "humantime_serde", default = "defaults::message_timeout")]
-    pub message_timeout: Duration,
-    #[serde(default = "defaults::max_message_length")]
-    pub max_message_length: usize,
-    #[serde(default = "defaults::max_note_length")]
-    pub max_note_length: u64,
+    pub server_location: String,
```

Comment:
```suggestion
    pub server_uri: Uri,
```

I mentioned this in another comment but this might be convenient. I think `uri` is a better name for this either way just to indicate what should be in there.